packages/commands/generate/src/index.ts

```diff
@@ -147,9 +147,20 @@ export const plugin: CliPlugin = {
           });
           const generateConfig: GenerateConfig = await config.extension('generate');
 
-          if (!generateConfig) {
+          if (!generateConfig && generateConfig.folders) {
             throw new Error(`You should provide a valid 'generate' config to generate schema from data model`);
           }
+          
+          if (!generateConfig.folders) {
+            throw new Error(`'generate' config missing 'folders' section that is required`);
+          }
+          
+          if (!generateConfig.graphqlCRUD) {
```

Comment:
I think we don't need to have this validation in case of `graphql generate --db`. So maybe it'd better to move this if `backend` or `client` is asked. What do you think?